airflow-core/tests/unit/api_fastapi/execution_api/versions/head/test_task_instances.py

```diff
@@ -1222,37 +1222,6 @@ def test_get_start_date_not_found(self, client):
         response = client.get(f"/execution/task-reschedules/{ti_id}/start_date")
         assert response.json() is None
 
-    def test_get_start_date_with_try_number(self, client, session, create_task_instance):
```

Comment:
We should keep this test -- although probably in `airflow-core/tests/unit/api_fastapi/execution_api/versions/head/test_task_reschedules.py`

Reply:
It's not possible to get this data for anything other than the current try number as we delete the old try data in `TI.prepare_db_for_next_try`.

Reply:
aah yeah we have other tests in TestGetRescheduleStartDate, sg